src_kt/javatests/arcs/stringEncoder/BUILD

```diff
@@ -11,7 +11,7 @@ arcs_kt_jvm_test_suite(
     name = "stringEncoder",
     package = "arcs.stringEncoder",
     deps = [
-        "//src/wasm/kotlin:arcs",
+        "//src/wasm/kotlin/java/arcs",
```

Comment:
This seems surprising to me i.e. that we have a java dir inside our kotlin dir.

Would be interested in the meaning of this, but am not against it, just don't understand.

Reply:
Yeah it's not ideal. Just want to get the copybara import to work